spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
@gibson042 Exactly. `raw` in fact is not needed when you have `directive` boolean flag which would simply state that 1) this `ExpressionStatement` contains string `Literal` inside of it and 2) `Literal.value` contains valid (unescaped) directive.

Why do you think it wouldn't work in this case? They wouldn't be identical since second example will not have `directive` flag (implicit `false`).